templates/.gitignore.tpl

```diff
@@ -61,6 +61,13 @@ Pulumi.*.yaml
 # Documentation output
 /apidocs
 
+### Start ignores inserted by other modules
+{{- $extraHook := (stencil.GetModuleHook "extraIgnores") }}
```

Comment:
Module hooks aren't per file, so we try to namespace them.

```suggestion
{{- $extraHook := (stencil.GetModuleHook "gitIgnore/extraIgnores") }}
```

Reply:
seems apply hook must specify source file though. and your past usage in stencil-circleci don't use namespace either, is it really necessary?

Reply:
just curious, not a big deal for adding namespace 
Here is your past usage
<img width="443" alt="image" src="https://user-images.githubusercontent.com/6810045/225385259-0d1de48c-8be8-454b-b77d-eaaf691c3dc5.png">

Reply:
added the namespace anyway to further decreasing the chance although the name is already very specific in the repo.